decidim-forms/app/views/decidim/forms/admin/questionnaires/edit.html.erb

```diff
@@ -1,8 +1,14 @@
-<% if (defined? Decidim::Templates) && choose_template? %>
+<% templates_defined = (defined? Decidim::Templates::Admin::Concerns::Templatable) %>
+
+<% if templates_defined && choose_template? %>
   <%= render partial: "decidim/templates/admin/questionnaire_templates/choose", locals: { target: questionnaire, form_title: t("decidim.forms.admin.questionnaires.edit.title") } %>
 <% else %>
   <%= decidim_form_for(@form, url: update_url, method: :put, html: { class: "form edit_questionnaire" }) do |form| %>
-    <%= render partial: "decidim/forms/admin/questionnaires/form", object: form, locals: { title: t("form.title", scope: "decidim.templates.admin.questionnaire_templates") } %>
+    <%= render partial: "decidim/forms/admin/questionnaires/form",
+               object: form,
+               locals: {
+                 title: templates_defined ? t("form.title", scope: "decidim.templates.admin.questionnaire_templates") : t(".title")
```

Comment:
Maybe this could be in the `HasQuestionnaire` concern, in a method like `form_title`

Reply:
I think that if its possible, could you make this conditional in a method like @microstudi says? Thanks!